guides/developers/event-display.md

```diff
@@ -56,42 +57,62 @@ Currently the sub managers are not big enough to be divided into multiple parts.
 
 ## [`UIManager`](../../packages/phoenix-event-display/src/managers/ui-manager/index.ts)
 
-The `UIManager` is responsible for all the UI aspects of the `EventDisplay`. This includes the Phoenix menu, dat.GUI menu, stats UI etc.
+The `UIManager` is responsible for all the UI aspects of the `EventDisplay`. This includes the Phoenix menu, dat.GUI menu, stats UI, etc.
 
 It takes care of tasks like:
 
 * Adding configuration options to Phoenix UI on loading geometry or event data.
-* Setting the UI color theme to dark or light
-* Applying clipping to geometries
-* Linking UI menu options with the `EventDisplay`
+* Setting the UI color theme to dark or light.
+* Applying clipping to geometries.
+* Linking UI menu options with the `EventDisplay`.
+* Configuring different color options for event-data.
 * etc.
 
 The `UIManager` is further divided into the following managers:
 
+* [**`ColorOptionsUI`**](../../packages/phoenix-event-display/src/managers/ui-manager/color-options.ts)  
+  Contains configurable color options with functions to color event data.
 * [**`DatGUIMenuUI`**](../../packages/phoenix-event-display/src/managers/ui-manager/dat-gui-ui.ts)   
   Contains functions for setting up all the options in dat.GUI menu.
 * [**`PhoenixMenuUI`**](../../packages/phoenix-event-display/src/managers/ui-manager/phoenix-menu/phoenix-menu-ui.ts)  
   Contains functions for setting up all the options in Phoenix menu.
+  
+It also has [**`PhoenixMenuConfigTypes`**](../../packages/phoenix-event-display/src/managers/ui-manager/phoenix-menu/config-types.ts) which provides types for Phoenix menu configuration.
 
 ### [`PhoenixMenuNode`](../../packages/phoenix-event-display/src/managers/ui-manager/phoenix-menu/phoenix-menu-node.ts)
 
 The `PhoenixMenuNode` is a class that maintains all the options of Phoenix menu at the program level. It doesn't implement the UI for Phoenix menu but contains all the logic.\
 It is designed to be adaptable to custom UIs. This class can be used to create a custom Phoenix menu like interface in any kind of frontend framework.
 
-For an overview, it contains functions like:
+For an overview, it contains functions for:
 
-* Adding children (of the same `PhoenixMenuNode` type) to a node
+* Adding children (of the same `PhoenixMenuNode` type) to a node.
 * Adding custom configuration of different types (like `checkbox`, `slider`, `button`, `label` etc.) to a node.
-* Getting or loading the state of a node
-* Removing self node or child nodes
-* etc.
+* Getting and loading the state of a node.
+* Removing a single child node, current node, or all child nodes.  
+* Finding a node in tree by name, or create one if not found.
+* etc.  
+
+### [`PhoenixUI`](../../packages/phoenix-event-display/src/managers/ui-manager/phoenix-ui.ts)  
```

Comment:
I think `PhoenixUI` should come before mentioning its implementations `DatGUIMenuUI` and `PhoenixMenuUI`.

Reply:
done. ✅